### ISSUE TITLE
Snowflake Release 7.22 causing issues with Warehouse deployment

### ISSUE THREAD

**Issue by brettwatkins-Built** [Snowflake-Labs/terraform-provider-snowflake#1930]
**Provider Version**

0.67.0

**Terraform Version**

1.4.6

**Snowflake Version**

7.22.1

**Describe the bug**

When deploying a new warehouse, I'm running into this error:

`Error: 001423 (22023): SQL compilation error:
 invalid property 'QUERY_ACCELERATION_MAX_SCALE_FACTOR'; feature 'Query Acceleration Service' not enabled`

**Expected behavior**

I would expect the warehouse to deploy without issue. The same code was working for Snowflake version 7.21 leading us to believe this is an issue with the most recent snowflake release (they don't even have release notes out as of 7/6/23) and the provider.  

**Code samples and commands**

```
resource "snowflake_warehouse" "example" {
  provider = snowflake.sysadmin
  name = local.warehouse_name
  auto_resume = true
  auto_suspend = 60
  min_cluster_count = var.env == "prod" ? 1 : null
  max_cluster_count = var.env == "prod" ? 3 : null
  initially_suspended = true
  warehouse_size = "xsmall"
  enable_query_acceleration = false
  query_acceleration_max_scale_factor = 0

  lifecycle {
    ignore_changes = [initially_suspended, query_acceleration_max_scale_factor]
  }
}
```

**Comment by CrownBerry** [comment]
Hi @brettwatkins-Built 
This problem was fixed in #1866 and should appear in the next release (0.67.1), as far as I know.

**Comment by sfc-gh-ngaberel** [comment]
v0.68 is out with @CrownBerry 's fix, closing this for now, let us know if the problem persists in the new version.
